source/javascripts/controllers/_ProvProfileController.js.erb

```diff
@@ -152,6 +152,10 @@ angular.module("BitriseWorkflowEditor").controller("ProvProfileController", func
 		return _.any(certificateService.certificates, function(aCertificate) {
 			return _.any(aCertificate.details, { serial: identitySerial });
 		});
+	}
+
+	viewModel.isIdentityExpired = function(identityExpiryDateTime) {
```

Comment:
Function name indicates that input argument will be identity. Either have it as an identity (and later on `identity.expireAt < new Date()`) or have a separate "is date expired" function in the date service, and use that function everywhere instead of having `viewModel.isIdentityExpired` defined.